tests/loop/011

```diff
@@ -33,7 +33,7 @@ test() {
 	errors=$(_dmesg_since_test_start | grep -c "operation not supported error, dev .*WRITE_ZEROES")
 
 	losetup --detach "$loop_dev"
-	umount "$TMPDIR/tmpfs"
+	umount -l "$TMPDIR/tmpfs"
```

Comment:
It is preferred to have long option names. I suggest to use "--lazy" instead of "-l".

Reply:
Thanks for the review, just updated.